api/sheets/get_name.md

```diff
@@ -1,46 +1,76 @@
 ---
 layout: default
-title: Node Name
+title: Names Associated with a DCID
 nav_order: 1
 parent: Google Sheets
 grand_parent: API
 ---
 
-# Get a node's name
+# Returning the Names Associated with a DCID
 
-Given a cell or a column range of cells with [DCIDs](/glossary.html#dcid) (unique identifiers for all entities in the Data Commons knowledge graph), return the names associated with the given DCIDs.
+The `=DCGETNAME(dcids)` formula returns the names associated with given [DCIDs](/glossary.html#dcid) to a cell or a column range of cells.
 
-## General information about this formula
+> **Note**
+> 
+> Be sure to follow the instructions for [Installing and Enabling the Sheets Add-On](/api/sheets/) before using this formula.
 
-**Formula**: `=DCGETNAME(dcids)`
+## Formula
 
-**Required arguments**:
+```
+=DCGETNAME(dcids)
+```
 
-* `dcids` - A list of nodes to query, identified by their DCIDs.
+## Required Arguments
+
+`dcids` - A list of node DCIDs to query
+
+## Returns
+
+The names associated with given node DCIDs to a cell or a range of cells.
+
+> **Note**:
+> It’s best to minimize the number of function calls to `=DCGETNAME(dcids)` by using a single call to get the names for a column of nodes. This is because a spreadsheet will make one call to a Google server [*per custom function call*](https://developers.google.com/apps-script/guides/sheets/functions#optimization). If your sheet contains thousands of separate calls to `=DCGETNAME(dcids)` you can expect it to be slow and return with errors.
 
 ## Examples
 
-Before trying this method out, make sure to follow the setup directions in [the main section for Sheets docs](/api/sheets/index.html).
+This section contains examples of returning the names associated with given DCIDs.
+
+### Example 1: Retrieve the Name of a Country by its DCID
+
+To retrieve the name of a country by its DCID, perform the following steps:
+
+1. Place your cursor in the cell where you want to add a DCID. In this case, cell A1.
+2. Click in the search box for the **Fill place dcids for selected cells** section. For this example, enter “Ivory Coast”. The cell is populated with a DCID of “country/CIV”.
```

Comment:
I don't understand what step 2 is trying to say. It's very weird to look up a DCID by name just to later query for that same name.

Reply:
i think that since this is the "first" api on the list, it's a way to tell users what to do with the "fill place dcid" searchbox. it does seem redundant though, and we already have such an example on the index page. let me drop this step then, and update the screenshot